lib/matplotlib/testing/decorators.py

```diff
@@ -482,7 +482,25 @@ def _image_directories(func):
     doesn't exist.
     """
     module_path = Path(sys.modules[func.__module__].__file__)
-    baseline_dir = module_path.parent / "baseline_images" / module_path.stem
+    if func.__module__.startswith("matplotlib."):
```

Comment:
Maybe not on this pass, but we should think about how packages can what package carries their test images.

Maybe entrypoints?